learn/validator-trustscore.md

```diff
@@ -66,66 +56,77 @@ $$
 
-<Callout type="info" no-title>
+<Callout type="info">
 
-The curves and constants presented in this document are subject to change at any time in the future. We will keep the community informed of any changes.
+The curves and constants presented in this document are subject to change at any time in the future. Any updates will be communicated to the community to ensure transparency.
 
 </Callout>
 
 ### Dominance
 
-The dominance factor ensures that no single validator controls too much of the network's total stake. If a validator controls a large portion of the total stake, they will get a lower score. We penalize validators with a higher stake compared to those with a lower stake, as a lower stake promotes a fair distribution of control across the network.
+The dominance factor ensures that no single validator controls an excessive portion of the network's total stake. Validators with a larger stake receive a lower score to encourage a fairer distribution of control across the network. This approach penalizes disproportionately large stakes while promoting diversity in validator participation.
 
 #### Dominance Ratio
 
-To find the dominance ratio ($s$) of a validator, we have two methods:
+The dominance ratio ($s$) of a validator is calculated using one of two methods, depending on the epoch's state. However, due to technical limitations, only the **Active Epoch Method** is currently implemented and used.
 
-1. **First method**: Calculate the ratio by dividing the validator's share by the total share of the network for an active epoch:
+**Active Epoch Method**:
+During an active epoch, the dominance ratio is determined by dividing the validator's share by the total network share:
 
 $$
 s = \frac{v}{Z}
 $$
 
-Where $v$ is the validator's share and $Z$ is the total network share. This method applies when the epoch is active, so we can access each validator's balance using the `getActiveValidators` function from the RPC.
+Where:
+- $v$: The validator's share.
+- $Z$: The total network share.
 
-2. **Second method**: A different approach is used for a closed epoch. This is less accurate due to some randomness and is considered a fallback option. Here we look at the slot distribution of each voting block, which reflects the amount staked by each validator. The dominance ratio is calculated by dividing the number of slots allocated to a validator by the total number of slots in that epoch:
+This method relies on the `getActiveValidators` function from the [RPC](https://www.nimiq.com/developers/build/set-up-your-own-node/rpc-docs/#getactivevalidators), which provides real-time balances of each active validator.
+
+**Closed Epoch Method**:
+For a closed epoch, a less precise fallback method exists, but is not currently implemented. This approach calculates the dominance ratio based on slot distribution across voting blocks. The ratio is derived by dividing the number of slots allocated to a validator by the total slots in the epoch:
```

Comment:
Actually, it is implemented. In the code we have:
- `dominanceRatioViaBalance`: This value is exact. The only drawback is that we can only fetch this value when the epoch is active
- `dominanceRatioViaRatio`: This value is an approximation and not exact. This approximation is really close

Here is a screenshot of the values we have in our database:

![image](https://github.com/user-attachments/assets/5b03da93-0790-420a-ac4b-44523a2d1f74)

Reply:
I can update this section of the docs if you don't have time or you are not sure. Let me know :)